config/schema/acquia_lift.settings.yml

```diff
@@ -98,19 +98,6 @@ acquia_lift.settings:
               type:
                 type: string
                 label: 'Item Type this is mapped to.'
-    thumbnail:
```

Comment:
D8 continues to work with obsolete config variables, be it a full obsolete variable or a partial obsolete within a variable. It has happened a few times so I know this for sure now.

I thought about writing an update hook, but then, I thought why pest our customers for no benefit? So I didn't. =D